vantiq-cloud-infra-operations/terraform_azure/new/env-dev/constants.tf

```diff
@@ -117,7 +98,6 @@ locals {
     dns_service_ip = "10.1.1.10"
 
     # network profile (optional)
-    docker_bridge_cidr = "172.17.0.1/16"
```

Comment:
Will be deprecated.
- https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/kubernetes_cluster#docker_bridge_cidr